db/seeds/prisons/PDI-portland.yml

```diff
@@ -20,10 +20,7 @@ recurring:
   tue:
   - 1345-1600
 unbookable:
-- 2018-12-25
-- 2019-01-01
+- 2019-12-27
```

Comment:
@Ed-Bajo - this date falls on a Friday, which is not a day the prison have recurring visits, so this whole section can be deleted